modules/ezcontent_node/modules/ezcontent_article/config/install/core.entity_form_display.node.article.default.yml

```diff
@@ -7,12 +7,14 @@ dependencies:
     - field.field.node.article.field_content
     - field.field.node.article.field_meta_tags
     - field.field.node.article.field_short_title
+    - field.field.node.article.field_social_sharing
     - field.field.node.article.field_subhead
     - field.field.node.article.field_summary
     - field.field.node.article.field_tags
     - field.field.node.article.field_thumbnail
     - field.field.node.article.field_yoast_seo
     - node.type.article
+    - workflows.workflow.default
```

Comment:
Do we need it ?

Reply:
This comes by default from the content_moderation module in core which defines the moderation workflow for Article and Basic page content types and we explicitly remove it every time.